filebot.install

```diff
@@ -11,11 +11,6 @@ fi
 }
 
 post_install() {
-  echo -e "\e[1;33m==>\e[0m Symlinking OpenJFX"
-
-  ln -sf /usr/lib/jvm/java-11-openjfx/lib/ /usr/share/filebot/openjfx
```

Comment:
This is not needed anymore, the package puts the files into `/usr/lib/jvm/java-17-openjdk/lib` and not an own directory.